editor/src/clj/editor/editor_extensions.clj

```diff
@@ -739,10 +739,11 @@
       (vreset! (:request-sync *execution-context*) true))
     file-path))
 
-(defn- find-resource [project evaluation-context path]
-  (some-> (project/get-resource-node project path evaluation-context)
-          (g/node-value :lines evaluation-context)
-          (data/lines-input-stream)))
+(defn- find-resource [project path]
+  (g/with-auto-evaluation-context evaluation-context
+    (some-> (project/get-resource-node project path evaluation-context)
+            (g/node-value :lines evaluation-context)
+            (data/lines-input-stream))))
```

Comment:
Editor scripts were keeping a reference around to a stale `evaluation-context` that was created at the time the editor scripts were reloaded. This was pretty broken, but I'm not sure this function was used a lot.